recipes/AffjaxPostNode/README.md

```diff
@@ -0,0 +1,21 @@
+# AffjaxPostNode
+
+Performs a simple HTTP Post request using Affjax library.
+
+The post data that is sent with the request makes use of argonaut to reduce boilerplate
+
+NB: this recipe depends upon the continuing availability of the dummy JSON provider "http://jsonplaceholder.typicode.com/posts"
+
+## Expected Behavior:
+
+Prints to the console:
+
+`-- POST http://jsonplaceholder.typicode.com/posts response: {"userId":22,"title":"title","id":101,"body":"body"}`
+
+## Dependencies used
+
+[xhr2](https://www.npmjs.com/package/xhr2)
```

Comment:
Can you clarify that this is only needed if using `affjax` in a Node environment?

Reply:
should i be committing the changes to `package.json` and `package-lock.json` for the `xhr2` dependency, btw?

i may have missed it but didn't see in the contribute file whether that's desired or not...

Reply:
You'll just need to add `xhr2` to [`package.json`](https://github.com/JordanMartinez/purescript-cookbook/blob/master/package.json). We're not tracking `package-lock.json`.
For example, you can see the `big-integer` dependency from the [BigIntJs recipe](https://github.com/JordanMartinez/purescript-cookbook/tree/master/recipes/BigIntJs) listed in that file.
Feel free to add some additional clarifications to the [contributing docs section](https://github.com/JordanMartinez/purescript-cookbook/blob/master/CONTRIBUTING.md#all-recipes-that-use-the-same-npm-dependency-must-use-the-same-version-across-all-recipes) on this step.

Reply:
have added this, will PR the CONTRIBUTING.md in separate PR